### PR TITLE
 DOCAPI-6887: Description of skewPop, skewSamp, kurtPop, kurtSamp functions.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #5200

### CODE REVIEW

File: docs/en/query_language/agg_functions/reference.md [hunk 1]
```diff
@@ -301,6 +300,100 @@ GROUP BY timeslot
...
+`expr` — [Expression](../syntax.md#syntax-expressions) returning a number.
+
+**Returned value**
+
+The skewness of given distribution. Type — [Float64](../../data_types/float.md)
+
+**Example of Use**
+
+```sql
+SELECT skewPop( value ) FROM series_with_value_column
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]: why spaces near round brackets? (and below too)

File: docs/en/query_language/agg_functions/reference.md [hunk 2]
```diff
@@ -301,6 +300,100 @@ GROUP BY timeslot
...
+```
+
+## skewSamp
+
+Computes the [sample skewness](https://en.wikipedia.org/wiki/Skewness) for sequence.
+
+It represents an unbiased estimate of the skewness of a random variable, if the values passed to the function are a sample of this random amount.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]: "amount" looks out of place, maybe something like "...if passed values form it's sample"

File: docs/en/query_language/agg_functions/reference.md [hunk 3]
```diff
@@ -301,6 +300,100 @@ GROUP BY timeslot
...
+skewSamp( expr )
+```
+
+**Parameters**
+
+`expr` — [Expression](../syntax.md#syntax-expressions) returning a number.
+
+**Returned value**
+
+The skewness of given distribution. Type — [Float64](../../data_types/float.md)
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]: what is `n`?

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/query_language/agg_functions/reference.md (hunk 2 above)
Comment (hczhcz) [reply docs/en/query_language/agg_functions/reference.md]: This sentence comes from the description of `varSamp`. It would be good to update that one too.

Comment (BayoNet) [reply docs/en/query_language/agg_functions/reference.md]: Done.

**Pull request merged**